ajax/libs/jodit/package.json

```diff
@@ -10,18 +10,36 @@
     "rich",
     "jodit"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "MIT",
   "author": "Chupurnov <chupurnov@gmail.com> (http://xdsoft.net/)",
   "repository": {
     "type": "git",
     "url": "https://github.com/xdan/jodit.git"
   },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/xdan/jodit.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "jodit.min.js",
+          "jodit.min.css",
+          "jodit.es2018.min.js",
+          "jodit.es2018.min.css"
+        ]
+      }
+    ]
+  },
   "npmName": "jodit",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "build",
       "files": [
-        "*jodit*.+(css|js|map)"
+        "jodit.min.js",
```

Comment:
Using a glob of [`jodit*.+(css|js)` would include all those files](https://www.digitalocean.com/community/tools/glob?glob=jodit%2A.%2B%28css%7Cjs%29&tests=jodit.css&tests=jodit.es2018.css&tests=jodit.es2018.js&tests=jodit.es2018.min.css&tests=jodit.es2018.min.js&tests=jodit.js&tests=jodit.min.css&tests=jodit.min.js) or you could use a glob of [`jodit*.min.+(css|js)` to include only the minified files](https://www.digitalocean.com/community/tools/glob?glob=jodit%2A.min.%2B%28css%7Cjs%29&tests=jodit.css&tests=jodit.es2018.css&tests=jodit.es2018.js&tests=jodit.es2018.min.css&tests=jodit.es2018.min.js&tests=jodit.js&tests=jodit.min.css&tests=jodit.min.js).

Reply:
Added *jodit*.+(css|js|map)